### PR TITLE
Added Dog D

### PR DESCRIPTION
**Pull request by joakimskoog**
Third time's a charm, I guess

### CODE REVIEW

File: data/dog_d.json [hunk 1]
```diff
@@ -0,0 +1,184 @@
+{
+  "name": "¨Dog D",
+  "tagline": "ANNIVERSARY IMPERIAL STOUT",
```
Comment (sammdec) [inline data/dog_d.json:3]: Could you set this as title case with a fullstop at the end please :+1:

File: data/dog_d.json [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+      },
+	  {
+        "name": "Wheat Malt",
+        "amount": {
+          "value": 0.63,
+          "unit": "kilograms"
+        }
+      }
+    ],
+    "hops": [
```
Comment (sammdec) [inline data/dog_d.json:83]: Again same with the hops these dont look like the right ones

File: data/dog_d.json (hunk 1 above)
Comment (joakimskoog) [inline data/dog_d.json:3]: Title case on the name and a fullstop at the end of the tagline?

**Comment by sammdec** [comment]
Apart form those changes it all looks ace, thanks so much for adding :+1:

File: data/dog_d.json (hunk 2 above)
Comment (joakimskoog) [inline data/dog_d.json:83]: Hmm, they're totally wrong. I wonder why, probably just sloppy. I'll fix it right away!

Comment (sammdec) [inline data/dog_d.json:83]: No worries it happened to me a couple of times i knocked my mouse and it landed on another recipe :+1:

**Pull request closed without merging**